sql/rpl_slave.cc

```diff
@@ -2300,7 +2300,11 @@ int io_thread_init_commands(MYSQL *mysql, Master_info *mi) {
   int ret = 0;
   DBUG_EXECUTE_IF("fake_5_5_version_slave", return ret;);
 
-  sprintf(query, "SET @slave_uuid= '%s'", server_uuid);
+  sprintf(query,
```

Comment:
query being of size 256 should be already enough here, but snprintf wouldn't hurt nevertheless